src/Security/Group.php

```diff
@@ -377,6 +377,53 @@ public function collateAncestorIDs()
     }
 
     /**
+     * Check if the group is a child of the given group or any parent groups
+     *
+     * @param string|int|Group $group Group instance, Group Code or ID
+     * @return bool Returns TRUE if the Group is a child of the given group, otherwise FALSE
+     */
+    public function inGroup($group)
+    {
+        if (is_numeric($group)) {
+            $groupCheckObj = Group::get()->byID($group);
+        } elseif (is_string($group)) {
+            $SQL_group = Convert::raw2sql($group);
+            $groupCheckObj = Group::get()->filter(['Code' => $SQL_group])->first();
```

Comment:
Don't raw2sql when using filter api; It'll double encode.

```php
Group::get()->filter('Code', $group)->first();
```

is all you need